sql/src/main/java/org/apache/druid/sql/calcite/planner/QueryUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.planner;
+
+import org.apache.calcite.util.Pair;
+import org.apache.druid.sql.calcite.rel.DruidQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class for queries
+ */
+public class QueryUtils
+{
+
+  private QueryUtils()
+  {
+  }
+
+  /**
+   * Builds the mappings for queryColumn to outputColumn
+   * @param fieldMapping The field mappigns
+   * @param druidQuery The Druid query
+   * @return Mappings for queryColumn to outputColumn
+   */
+  public static List<ColumnMapping> buildColumnMappings(
```

Comment:
It'd be great if we can add a few UTs for this public method.

Reply:
talked offline, and decided UTs here wouldnt be very useful as this function is very simple, and already fully covered in other tests.